templates/repo/issue/view_content/comments.tmpl

```diff
@@ -233,5 +233,197 @@
+												</td>
+											</tr>
+
+
+											<tr class="same-code nl-1 ol-1">
+
+												<td class="lines-num lines-num-old">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0L27">27</span>
+												</td>
+												<td class="lines-num lines-num-new">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0R27">27</span>
+													<a class="ui green button add-code-comment add-code-comment-right" data-path="vendor/vendor.json" data-side="right" data-idx="27">+</a>
+												</td>
+
+												<td class="lines-code ">
+													<pre><code class="wrap language-json hljs"> 			<span class="hljs-string">"revisionTime"</span>: <span class="hljs-string">"2016-08-23T15:25:51Z"</span></code></pre>
+												</td>
+											</tr>
+
+
+											<tr class="same-code nl-2 ol-2">
+
+												<td class="lines-num lines-num-old">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0L28">28</span>
+												</td>
+												<td class="lines-num lines-num-new">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0R28">28</span>
+													<a class="ui green button add-code-comment add-code-comment-right" data-path="vendor/vendor.json" data-side="right" data-idx="28">+</a>
+												</td>
+
+												<td class="lines-code ">
+													<pre><code class="wrap language-json hljs"> 		}</code></pre>
+												</td>
+											</tr>
+
+
+											<tr class="same-code nl-3 ol-3">
+
+												<td class="lines-num lines-num-old">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0L29">29</span>
+												</td>
+												<td class="lines-num lines-num-new">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0R29">29</span>
+													<a class="ui green button add-code-comment add-code-comment-right" data-path="vendor/vendor.json" data-side="right" data-idx="29">+</a>
+												</td>
+
+												<td class="lines-code ">
+													<pre><code class="wrap language-json hljs"> 	],</code></pre>
+												</td>
+											</tr>
+
+											<tr class="del-code nl-4 ol-4">
+
+												<td class="lines-num lines-num-old">
+													<span rel="diff-d9bff1704388955a5d24ad2abc8aea5a312994d0L30">30</span>
+												</td>
+												<td class="lines-num lines-num-new">
+													<span rel=""></span>
+													<a class="ui green button add-code-comment add-code-comment-left" data-path="vendor/vendor.json" data-side="left" data-idx="30">+</a>
+												</td>
+
+												<td class="lines-code lines-code-old">
+													<pre><code class="wrap language-json hljs">-	<span class="hljs-string">"rootPath"</span>: <span class="hljs-string">"</span><span class="removed-code"><span class="hljs-string">github.com</span></span><span class="hljs-string">/JonasFranzDEV/drone-gitea-release"</span></code></pre>
```

Comment:
This hard-coded content does not belong here

Reply:
Just to mention: This comment was added via "comment" on the diff - I did not press any "review" button.
Just to have an example against your statement "GitHub only shows single comments in the diff view too"
But thanks for your work on this either way. I am really excited to see this merged to master :)

Reply:
This is some sort of placeholder as I stated somewhere before. It is used for ui prototyping. Currently I am searching a way to render the code efficently.